src/main/java/uk/gov/hmcts/reform/notificationservice/data/NotificationRepository.java

```diff
@@ -64,7 +64,8 @@ public NotificationRepository(
 
     public List<Notification> findPending() {
         return jdbcTemplate.query(
-            "SELECT * FROM notifications WHERE status = :status and confirmation_id IS NULL",
+            "SELECT * FROM notifications WHERE status = :status and confirmation_id IS NULL and "
+                + "created_at < (now()::timestamp - interval '2 hours')",
```

Comment:
this should be made configurable per environment

Reply:
done